### PR TITLE
Added the FreeCAD Stream Deck Addon module

### PR DESCRIPTION
**Pull request by Giraut**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -181,6 +181,10 @@
 	path = Freecad-Built-in-themes-beta
 	url = https://github.com/MisterMakerNL/Freecad-Built-in-themes-beta
 	branch = main
+[submodule "FreeCAD Stream Deck Addon"]
```
Comment (chennes) [inline .gitmodules:184]: The Addon Manager expects the submodule name entry to match, the path, so this should be "freecad_streamdeck_addon"

**Comment by chennes** [comment]
I've also submitted three issues to your repository for the things that should be changed in package.xml so that the Addon Manager reads it correctly.

**Comment by Giraut** [comment]
> I've also submitted three issues to your repository for the things that should be changed in package.xml so that the Addon Manager reads it correctly.

Thanks. They are fixed.

**Comment by chennes** [comment]
This will appear in the Addon Manager next time people recache. The addon stats from GitHub will be updated at around midnight CET and yours will be available then.

**Comment by Giraut** [comment]
Thanks!

**Comment by Giraut** [comment]
Hmm I see it, but there is one thing: I thought it had to be sorted alphabetically by path, but apparently it was by human readable name.

As a result, Streamdeck Addon (the human-readable name) appears between "freecad.gears workbench" and "GDML workbench" because the path is "freecad_streamdeck_addon".

Do you want me to submit a new PR to move it after "SteelColumn" where it should be?

**Comment by chennes** [comment]
Do you mean in the .gitmodules file?

**Comment by Giraut** [comment]
Yes. It's a bit confusing because the path "freecad_streamdeck_addon" is in the right place alphabetically, but what's displayed in the FreeCAD addon manager is the human-readable name "Stream Deck Addon", and that definitely isn't.

**Pull request merged**